packages/drivers/odsp-driver/src/index.ts

```diff
@@ -49,4 +49,13 @@ export {
 	storeLocatorInOdspUrl,
 } from "./odspFluidFileLink";
 
-export { parseCompactSnapshotResponse } from "./compactSnapshotParser";
+export {
+	IOdspCache,
+	IPersistedFileCache,
+	INonPersistentCache,
+	IPrefetchSnapshotContents,
+} from "./odspCache";
+export { ICacheAndTracker, EpochTracker, FetchType, FetchTypeInternal } from "./epochTracker";
```

Comment:
Since `EpochTracker` was not already exported by this library (and therefore no consumers are instantiating objects of that type), I would update this to be a type-only export (since presumably something else is referencing it, and that's why we needed to add it here at all).
```suggestion
export { ICacheAndTracker, type EpochTracker, FetchType, FetchTypeInternal } from "./epochTracker";
```